colour/models/rgb/transfer_functions/log.py

```diff
@@ -0,0 +1,142 @@
+#
+"""
+Log2 Shaper Implementation
```

Comment:
The title could be _Common Log Encodings_ or alike, what do you think @nick-shaw?

Reply:
Would make sense to keep it more general. This module could be expanded later to include things like the generic camera log function for CLF 3.0.